### PR TITLE
add label values for test modules

### PR DESCRIPTION
**Pull request by anniehedgpeth**
## Background

This branch adds labels (aka tags) to resources for the test modules and creates a variable for labels for the root module.

[Asana Task](https://app.asana.com/0/1181500399442529/1200356536468561/f)

## How Has This Been Tested

Planned locally. Will test here.

## This PR makes me feel

label everything

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -36,6 +36,12 @@ variable "ca_certificate_secret" {
   type        = string
 }
 
+variable "labels" {
+  type        = map(string)
+  description = "Labels to apply to resources"
+  default     = { "label" : "label" }
```
Comment (aaron-lane) [inline variables.tf:42]: I think that an empty default would be less surprising for users.

```suggestion
  default     = {}
```

File: main.tf [hunk 2]
```diff
@@ -36,7 +35,7 @@ module "object_storage" {
   source = "./modules/object_storage"
 
   namespace = var.namespace
-  labels    = local.labels
+  labels    = var.labels
```
Comment (aaron-lane) [inline main.tf:38]: Note for posterity: we need to ensure that all submodules either accept labels or don't need to.

**Review by aaron-lane — CHANGES_REQUESTED** [review]
Nice! Just one requested change below.

File: main.tf (hunk 2 above)
Comment (anniehedgpeth) [reply main.tf:38]: I double-checked and found one that I missed! Redis can receive labels, but the rest are good to go. I just added it.

**Review by aaron-lane — APPROVED** [review]

**Pull request merged**